recipes/conversation_view/feed_pill/feed_item_pill.stories.js

```diff
@@ -57,45 +63,25 @@ const DefaultTemplate = (args) => createTemplateFromVueFile(
   DtRecipeFeedItemPillDefaultTemplate,
 );
 
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtRecipeFeedItemPillVariantsTemplate,
+);
```

Comment:
Remove argTypes here while moving to Vue 3 or the story might not work properly, check other stories for references.

```suggestion
const VariantsTemplate = (args) => createTemplateFromVueFile(
  args,
  DtRecipeFeedItemPillVariantsTemplate,
);
```

Reply:
Oh wow, didn't saw this one, thanks!